datafusion/src/physical_plan/expressions/average.rs

```diff
@@ -62,6 +62,24 @@ pub fn avg_return_type(arg_type: &DataType) -> Result<DataType> {
     }
 }
 
+pub(crate) fn is_avg_support_arg_type(arg_type: &DataType) -> bool {
+    // TODO support the interval
+    // TODO: do we need to support the unsigned data type?
```

Comment:
What do you mean by `the unsigned data type`? If you mean "do we support AVG(UInt8)" I think the answer is yes.

Reply:
got it.